modules/apps/object/object-rest-api/build.gradle

```diff
@@ -2,6 +2,7 @@ dependencies {
 	compileOnly group: "com.fasterxml.jackson.core", name: "jackson-annotations", version: "2.13.4"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
 	compileOnly group: "io.swagger.core.v3", name: "swagger-annotations", version: "2.0.5"
+	compileOnly group: "io.swagger.core.v3", name: "swagger-models", version: "2.0.6"
```

Comment:
Is this needed¿

Reply:
Yes, as the package includes the Schema class